spec/inputs/syslog_spec.rb

```diff
@@ -31,6 +31,7 @@ def pattern_path(path)
   SYSLOG_LINE = "<164>Oct 26 15:19:25 1.2.3.4 %ASA-4-106023: Deny udp src DRAC:10.1.2.3/43434 dst outside:192.168.0.1/53 by access-group \"acl_drac\" [0x0, 0x0]"
 
   it "should properly handle priority, severity and facilities" do
+    skip 'elastic/logstash#11196 known LS 7.5 issue' if ENV['ELASTIC_STACK_VERSION'] && JRUBY_VERSION.eql?('9.2.8.0')
```

Comment:
using LS-like JOPTS triggered https://github.com/elastic/logstash/issues/11196 here as well
the fix is not yet in (7.x ->) 7.5 (only in the 7.x SNAPSHOT -> 7.6) so I setup excludes ...